spectral_cube/spectral_cube.py

```diff
@@ -2359,6 +2359,19 @@ def reproject(self, header, order='bilinear'):
             or an integer. A value of ``0`` indicates nearest neighbor
             interpolation.
         """
+
+        try:
+            from reproject.version import version
+        except ImportError:
+            raise ImportError("Requires the reproject package to be"
+                              " installed.")
+
+        # Need version > 0.2 to work with cubes
+        from distutils.version import LooseVersion
+        if LooseVersion(version) < "0.3":
+            raise Warning("Requires version >=0.3 of reproject. The current "
+                          "version is: {}".format(version))
```

Comment:
I think python >=2.7 supports "{}".format(...), but <2.7 didn't.... if it fails, this is a place to look (then use "{0}".format() instead)